tools/ports/libmodplug.py

```diff
@@ -24,14 +24,13 @@ def create(output_path):
     ports.clear_project_build('libmodplug')
 
     source_path = os.path.join(ports.get_dir(), 'libmodplug', 'libmodplug-' + TAG)
-    dest_path = os.path.join(ports.get_build_dir(), 'libmodplug')
+    src_dir = os.path.join(source_path, 'src')
+    libmodplug_path = os.path.join(src_dir, 'libmodplug')
 
-    sauce_path = os.path.join(dest_path, 'src')
```

Comment:
hahaha what a great variable name. Sad that we're deleting it.

Reply:
I know!  I thought it was a typo at first..  :)